src/helm/values.yaml

```diff
@@ -0,0 +1,110 @@
+# -- route
```

Comment:
As stated IRL, we suggest to:

- remove the `joanie` prefix in our variable names
- move variables to camelCase as suggested by Helm good practices